.github/workflows/build.yml

```diff
@@ -50,3 +50,22 @@ jobs:
       - name: Doctest
         run: |
           make doctest -C docs/
+      - name: Send mail
+        # Notify when cron job fails
+        if: (github.event_name == 'schedule' && failure())
+        uses: dawidd6/action-send-mail@v2
+        with:
+          # mail server settings
+          server_address: smtp.gmail.com
+          server_port: 465
+          # user credentials
+          username: ${{ secrets.NOTIFY_EMAIL }}
```

Comment:
I've currently created a new gmail account notificationsvivarium@gmail.com that sends the email.

Reply:
I believe I would need to regenerate the password since I didn't save it and would need to do this for all repositories we want it in.

Reply:
After discussions today, I will update the secrets email to be the vivarium dev email when we want to implement this in all repositories as that will be a sweeping change.

Reply:
Did you update the email to vivarium.dev@gmail.com? If not, I think you should do that now.

Reply:
Yes I have - this currently won't work because I will need to set up the secrets password with the the app password in that email account through gmail.